source/cloud/cloud-integrations/cloud-bot-accounts.rst

```diff
@@ -93,9 +93,13 @@ What happens if a plugin is using a bot account that already exists as a user ac
 
 For a concrete example, suppose you enable the `Mattermost GitHub plugin <https://github.com/mattermost/mattermost-plugin-github>`_, which uses a ``github`` bot account, while an existing ``github`` user account was created for webhook integrations.
 
-Once the plugin is enabled, the plugin posts as the ``github`` account but without a `BOT` tag. An error message is logged to the server logs recommending the System Admin to convert the ``github`` user to a bot account by running ``mattermost user convert <username> --bot`` in the CLI.
+Once the plugin is enabled, the plugin posts as the ``github`` account but without a `BOT` tag. An error message is logged to the Mattermost logs recommending the System Admin to convert the ``github`` user to a bot account by using ``mmctl``:
```

Comment:
Given that in Cloud admins don't have access to the log, wondering if mentioning we log an error messsage in the logs adds any value here. So maybe it's worth just stating that they should convert the user to bot directly.

Reply:
Agreed and great catch, @marianunez. I'll remove the reference to the server logs.